packages/medusa-payment-klarna/src/services/klarna-provider.js

```diff
@@ -246,29 +246,28 @@ class KlarnaProviderService extends PaymentService {
 
   /**
    * Status for Klarna order.
-   * @param {Object} paymentData - payment method data from cart
-   * @returns {string} the status of the Klarna order
+   * @param {PaymentSessionData} paymentSessionsData - payment session data from cart
+   * @returns {Promise<PaymentSessionStatus>} the status of the Klarna order
    */
-  async getStatus(paymentData) {
-    const { order_id } = paymentData
+  async getStatus(paymentSessionsData) {
```

Comment:
very, very nit(non-blocking): Should be in singular form
```suggestion
  async getStatus(paymentSessionData) {
```